processor/error/package_tests/TestProcessErrorUserAgentAndIP.approved.json

```diff
@@ -0,0 +1,31 @@
+{
+    "events": [
+        {
+            "@timestamp": "2017-05-09T15:04:05.999999Z",
+            "context": {
+                "service": {
+                    "agent": {
+                        "name": "python",
+                        "version": "1.0"
+                    },
+                    "name": "1234_service-12a3"
+                },
+                "user": {
+                    "ip": "127.0.0.1",
+                    "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_5)",
```

Comment:
I still find it confusing to have the `user-agent` at several places now, and do not quite understand why it is placed within `user`.
I'd rather go for a neutral place, like e.g. `context.user-agent`, depending on the actual agent it would be clear if it concerns an end user or not, but I know that there are lots of discussions and different opinions about that.